riptide/webhook.py

```
@@ -468,6 +468,13 @@ async def handle_issue_comment(payload: dict, delivery_id: str) -> Response:
                     client.post_pr_comment(
                         installation_id, owner, repo_name, pr_number, result
                     )
+                    # After responding, try to drain the queue if a fix just freed up
+                    if "queued" in (result or ""):
+                        try:
+                            from riptide.fixer import process_fix_queue
+                            process_fix_queue(client, owner, repo_name)
+                        except Exception as e:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)